### PR TITLE
[REFACTORING] Force absolute path to extra properties file and cassan…

### PR DESCRIPTION
**Pull request by Arsnael**
…dra driver conf file for docker images

Note I need a bit more time to test this though...

### CODE REVIEW

**Review by chibenwa — APPROVED** [review]

File: tmail-backend/apps/distributed-es6-backport/pom.xml [hunk 1]
```diff
@@ -414,6 +414,9 @@
                             <port>8000</port> <!-- WEBADMIN -->
                         </ports>
                         <appRoot>/root</appRoot>
+                        <jvmFlags>
+                            <jvmFlag>-Dextra.props=/root/conf/jvm.properties</jvmFlag>
```
Comment (Arsnael) [inline tmail-backend/apps/distributed-es6-backport/pom.xml:418]: Thoughts: Do you think it's necessary? or do we add that in the helm instead directly?

Comment (chibenwa) [reply tmail-backend/apps/distributed-es6-backport/pom.xml:418]: this can avoid bad surprises when running docker directly so imo lets keep it

**Comment by Arsnael** [comment]
```
10:57:53.231 [ERROR] 🐳 [linagora/tmail-backend-distributed:latest] - Log output from the failed container:
Exception in thread "main" org.apache.james.server.core.MissingArgumentException: Server needs a working.directory env entry
	at com.linagora.tmail.james.app.DistributedJamesConfiguration$Builder.useWorkingDirectoryEnvProperty(DistributedJamesConfiguration.java:56)
	at com.linagora.tmail.james.app.DistributedServer.main(DistributedServer.java:235)
```
Does it need all the jvm flags declared like in apache james? why if I declare one for extra props it wants a working directory, while if you don't declare any jvm flags it runs without asking? Not sure to get it

**Review by chibenwa — APPROVED** [review]

**Comment by Arsnael** [comment]
```
[INFO] Running com.linagora.tmail.deployment.DistributedImapAndSmtpTest
[ERROR] Tests run: 3, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 470.33 s <<< FAILURE! - in com.linagora.tmail.deployment.DistributedImapAndSmtpTest
[ERROR] validateDeploymentWithMailsFromSmtp  Time elapsed: 151.771 s  <<< ERROR!
org.awaitility.core.ConditionTimeoutException: Condition with org.apache.james.mpt.imapmailbox.external.james.DeploymentValidation was not fulfilled within 10 seconds.
```
second time... likely an issue

**Review by quantranhong1999 — APPROVED** [review]

**Comment by Arsnael** [comment]
I don't know I'm removing those jvmFlags... for some reasons the distributed deployment tests don't seem to like it. I could not find really the reason why though :(

**Comment by chibenwa** [comment]
>  for some reasons the distributed deployment tests don't seem to like it. I could not find really the reason why though :(

No logs in the James containers?

**Comment by Arsnael** [comment]
> > for some reasons the distributed deployment tests don't seem to like it. I could not find really the reason why though :(
> 
> No logs in the James containers?

Forgot... Honestly feel like I've wasted enough time on this already :(

**Comment by Arsnael** [comment]
cd #474

**Pull request closed without merging**